pkgs/top-level/all-packages.nix

```diff
@@ -19527,6 +19527,8 @@ with pkgs;
 
   libtool_2 = callPackage ../development/tools/misc/libtool/libtool2.nix { };
 
+  glibtool = callPackage ../development/tools/misc/libtool/glibtool.nix { };
```

Comment:
Please move the package to `pkgs/by-name/gl/glibtool/package.nix` and remove this line, thanks.

Reply:
Done.